pgcli/main.py

```diff
@@ -692,68 +682,54 @@ def prompt_tokens(_):
                     len(prompt) > self.max_len_prompt):
                 prompt = self.get_prompt('\\d> ')
 
-            return [(Token.Prompt, prompt)]
```

Comment:
In mycli branch this was rewitten as `return [('class:prompt', prompt)]` I don't know if that would be better.

Reply:
👍